lib/worker/app.js

```diff
@@ -3,10 +3,17 @@
 //        so that method is more easily accessible.
 
 const { Probot } = require('probot');
-const { getPrivateKey } = require('@probot/get-private-key');
+const { AppSecrets , Installation} = require('../models');
 
-module.exports = new Probot({
-  appId: process.env.APP_ID,
-  privateKey: getPrivateKey(),
-  logLevel: process.env.LOG_LEVEL,
-});
+
+module.exports = async (jiraHost) => { 
+  const installation = await Installation.getForHost(jiraHost);
+  const host = installation.githubHost;  
+  const ghaeInstanceData = await AppSecrets.getForHost(host); 
+
+  return new Probot({
+    appId: ghaeInstanceData.appId,
```

Comment:
add null check for ghaeInstanceData. and if its null then set from env file. so that it will work with github.com as well

`(ghaeInstanceData && ghaeInstanceData.appId) || process.env.APP_ID,`